packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -124,6 +136,25 @@ export class FocusTrapZone extends BaseComponent<IFocusTrapZoneProps, {}> implem
     }
   }
 
+  private _onFocusCapture = (ev: React.FocusEvent<HTMLDivElement>) => {
+    if (this.props.onFocusCapture) {
+      this.props.onFocusCapture(ev);
+    }
+
```

Comment:
Consider checking if the default has been prevented after this if check and bail out if it has been prevented

Reply:
I don't think that's the semantic purpose of preventDefault().  It is meant to prevent default browser actions.

Here is the w3.org spec section that specifies common event actions. These are the actions that preventDefault() would "prevent", which vary for each element type: https://www.w3.org/TR/html5/sec-forms.html#common-event-behaviors

Reply:
Yes, in the most pure sense you're right, but if someone passes in a focusCapture handler and they preventDefault, they likely did what they wanted when this event fires and anything that you do on top of it may result in unexpected behavior for them which they now have no control over.

For this reason, there's isDefaultPrevented so that javascript logic can tell if someone has already wanted to preventDefault and decided not to act on the event.